contracts/AgentMech.sol

```diff
@@ -156,45 +217,73 @@ contract AgentMech is ERC721Mech {
         // Increase the total number of requests
         numTotalRequests++;
```

Comment:
The request amount statistics is recorded per mech, and per total on mech marketplace level.